src/org/zaproxy/zap/extension/selenium/PopupMenuItemOpenInBrowser.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2017 The ZAP Development Team
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.selenium;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.view.popup.PopupMenuItemHttpMessageContainer;
+
+public class PopupMenuItemOpenInBrowser extends
+        PopupMenuItemHttpMessageContainer {
+
+    private static final long serialVersionUID = 1L;
+
+    private Logger logger = Logger.getLogger(PopupMenuItemOpenInBrowser.class);
+    private ExtensionSelenium ext;
+    private ProvidedBrowser browser;
+
+    /**
+     * @param label
+     * @param ext
+     * @param ext
+     * @param browser
+     */

```

Comment:
This JavaDoc should be completed or removed. (If completed, one of the ext params should be dropped.)